examples/olympic.js

```diff
@@ -17,6 +17,10 @@ function ingest_profile(hatch, uri) {
         const modified_date = new Date(Date.parse(modified_str));
         asset.set_last_modified_date(modified_date);
 
+        // Pull out the description.
+        const description = $profile('meta[name="description"]').attr('content');
+        asset.set_synopsis(description);
```

Comment:
Oh - sorry about that I confused this with the other (testing) comment. If we can fix the other examples and add this field, that would be best. We can open stories for them but that would just waste  a ton of time getting it done.